stdlib/Unicode/src/Unicode.jl

```diff
@@ -89,4 +89,44 @@ letter combined with an accent mark is a single grapheme.)
 """
 graphemes(s::AbstractString) = Base.Unicode.GraphemeIterator{typeof(s)}(s)
 
+const EMOJI_RANGES = [
+    0x1F600:0x1F64F,  # Emoticons
+    0x1F300:0x1F5FF,  # Misc Symbols and Pictographs
+    0x1F680:0x1F6FF,  # Transport and Map
+    0x2600:0x26FF,    # Misc symbols
+    0x2700:0x27BF,    # Dingbats
+    0xFE00:0xFE0F,    # Variation Selectors
```

Comment:
```suggestion
    0x02600:0x026FF,  # Misc symbols
    0x02700:0x027BF,  # Dingbats
    0x0FE00:0x0FE0F,  # Variation Selectors
```
Just for nicer alignment.